src/lib.rs

```diff
@@ -150,10 +190,17 @@
 #![warn(missing_docs)]
 #![cfg_attr(feature = "nightly", feature(panic_handler))]
 
+#![cfg_attr(all(not(feature = "use_std"), not(test)), no_std)]
```

Comment:
Sorry, I was being stupid with namespaces. I've moved the `extern crate
std` into the `tests` module.

On Thu, Jan 14, 2016 at 10:21 PM, Alex Crichton <notifications@github.com>
wrote:

> In src/lib.rs
> <https://github.com/rust-lang-nursery/log/pull/70#discussion_r49794509>:
>
> > @@ -150,10 +190,17 @@
> >  #![warn(missing_docs)]
> >  #![cfg_attr(feature = "nightly", feature(panic_handler))]
> >
> > +#![cfg_attr(all(not(feature = "use_std"), not(test)), no_std)]
>
> I mean putting the crate in the module it's needed, not up here. This is a
> core crate of the Rust ecosystem and is likely to be used as an example for
> downstream code (e.g. idioms and such), so I'd prefer to have it as
> idiomatic as possible.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rust-lang-nursery/log/pull/70/files#r49794509>.
>